code/race/racing.cpp

```diff
@@ -19,20 +25,19 @@ int main() {
       }
     };
 
-    // Run with two threads
-    std::thread t1(inc100);
-    std::thread t2(inc100);
-    for (auto t : {&t1,&t2}) t->join();
+    // Start up all threads:
+    std::vector<std::thread> threads;
+    for (unsigned int i = 0; i < nThread; ++i) {
+      threads.emplace_back(inc100);
+    }
+    for (auto & thread : threads) thread.join();
```

Comment:
Haha, I looked at your code and thought: This is just:
```suggestion
    std::vector<std::jthread>(nThread, inc100);
```
But I am not sure if this is more readable :D

Reply:
It doesn't compile AFAIK, since you have to pass a thread and not the lambda to the constructor.
Be my guest:
https://godbolt.org/z/qKM76rosE

Reply:
Or phrased differently: I don't know a way to fill the vector other than emplacing or moving the threads.

Reply:
Right, vector's ctor expects its T as the ctor argument, and not a type convertible to T. Got me :)